openretina/insilico/tuning_analyses/response_gradient.py

```diff
@@ -0,0 +1,110 @@
+from torch import optim
+from torch import nn
+from copy import deepcopy
+import torch
+import numpy as np
+from openretina.insilico.stimulus_optimization.objective import SingleNeuronObjective
+
+class MeiAcrossContrasts(nn.Module):
+    """
+    PyTorch Module wrapper around a stimulus to allow tracking response gradients onto the stimulus.
+    """
+    def __init__(self, contrast_values: torch.TensorType, stim: torch.TensorType):
+        """
+        Initialize the ResponseGradient class.
+
+        Parameters:
+        contrast_values (torch.TensorType): A torch Tensor with 2 scalar values that the MEI green and UV channels will be multiplied with.
+        stim (torch.TensorType): MEI stimulus as torch Tensor.
```

Comment:
```suggestion
        contrast_values (torch.Tensor): A torch Tensor with 2 scalar values that the MEI green and UV channels will be multiplied with.
        stim (torch.Tensor): MEI stimulus as torch Tensor.
```
Does it have to be an MEI stimulus or can it be any stimulus?